projects/plugins/jetpack/extensions/shared/external-media/media-category/index.ts

```diff
@@ -0,0 +1,203 @@
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+
+// Pexels constants
+const PEXELS_ID = 'pexels';
+const PEXELS_NAME = 'Pexels Free Photos';
+const PEXELS_SEARCH_PLACEHOLDER = `Search ${ PEXELS_NAME }`;
+const DEFAULT_PEXELS_SEARCH: MediaSearch = {
+	per_page: 25,
+	search: 'mountain',
+};
+
+// Google Photos constants
+const GOOGLE_PHOTOS_ID = 'google_photos';
+const GOOGLE_PHOTOS_NAME = 'Google Photos';
+const GOOGLE_PHOTOS_SEARCH_PLACEHOLDER = `Search ${ GOOGLE_PHOTOS_NAME }`;
+const DEFAULT_GOOGLE_PHOTOS_SEARCH: MediaSearch = {
+	per_page: 25,
+	search: '',
+};
+
+/**
+ * External media endpoints.
+ */
+// eslint-disable-next-line no-shadow
```

Comment:
What is disabling this line? The linter does not return any error for me if I remove this line.

Reply:
I can't commit if I remove that line, GH hook is giving me an error about shadow vars. I'll try again just in case.

Reply:
Nope, I need to disable that check otherwise I can't commit.

Reply:
Seems this is caused by a TS misconfig according to https://stackoverflow.com/questions/63961803/eslint-says-all-enums-in-typescript-app-are-already-declared-in-the-upper-scope